src/UserManual.php

```diff
@@ -189,24 +188,6 @@ public function getSettings(): ?Model
             $settings->$settingName = $settingValueOverride ?? $settingValue;
         }
 
-        // Allow handles from config
```

Comment:
@johnfmorton was removing this code block here necessary? I think having this is what allows the default template to work

Reply:
As mentioned above, I'm submitting a new pull request that restore the ability to set the section by a handle string in addition to a section ID int.